conda/recipes/librmm/recipe.yaml

```diff
@@ -72,40 +74,72 @@ outputs:
       ignore_run_exports:
         by_name:
           - cuda-version
+          - if: cuda_major == "11"
+            then: cudatoolkit
```

Comment:
I guess the nvcc (CUDA 11) compiler [issues this run-export on `cudatoolkit`](https://github.com/conda-forge/nvcc-feedstock/blob/6de62063c45045de06e982ae3d6c536a4dc1c1ef/recipe/meta.yaml#L23). Is it overconstraining the version (like >=11.8)? Or is it just redundant with our existing `run:` dependency on `cudatoolkit` for CUDA 11?

Reply:
it does overconstrain:
```
>=11.8.0,<12.0a0 (RE of [cache-build: nvcc_linux-64])
```